packages/aws-cdk/test/integ/cli/test.sh

```diff
@@ -7,9 +7,35 @@ header CLI Integration Tests
 
```

Comment:
Missing comments around the code. This is hairy enough...

Reply:
I was hoping the code was self documenting :(

Reply:
added